provider/idps/htpasswd.go

```diff
@@ -39,3 +43,70 @@ func CreateHTPasswdIDPBuilder(ctx context.Context, state *HTPasswdIdentityProvid
 	}
 	return builder
 }
+
+func HTPasswdValidators() []tfsdk.AttributeValidator {
```

Comment:
Can't htpasswd and admincreds use the same validator?
Istead of having this duplicated?

Reply:
As of now, it can. They are the same. But we do plan to extend the htpasswd IDP (soon) to also accept a list of users. This will change the validator, and it won’t apply to the admin case.